### PR TITLE
Nr/troubleshooting new

### PR DESCRIPTION
**Pull request by nithyaramakrishnan**
Hi @smguilia Shawna, 

Please see new changes to the "cause" and "resolution steps" for the troubleshooting section :"SGX driver failure". Hope it is clear now. 

Thanks,
Nithya

### CODE REVIEW

File: ts-index.md [hunk 1]
```diff
@@ -216,9 +216,49 @@ To resolve the issue, delete the following CRDs.
...
+
+{: tsCauses}
+You might have not installed the intel/sgx libraries such as `libsgx-enclave-common` because the worker node is in a bad state such as the `dpk` package was not working properly or the installer pod was not installing correctly on the said worker node.
+
+{: tsResolve}
+To resolve the issue, you can use the following steps:
+
+1. Access the worker node on the cluster and restart it.
+
+2. Reinstall the SGX driver. This will run the `datashiled-sgx` pod again and install the necessary packages.
```
Comment (smguilia) [inline ts-index.md]: If they don't uninstall the corrupted package before they try to reinstall the driver will the reinstall still work?
@nithyaramakrishnan

Comment (smguilia) [reply ts-index.md]: Maybe something like this?

{: tsCauses}
You might have a library, such as `libsgx-enclave-common`, that is incorrectly installed.

{: tsResolve}
To resolve the issue, try uninstalling the library and then reinstalling the driver.

1. Access the worker node on your cluster and restart it.
2. Uninstall the library.

  ```
  dpkg --purge libsgx-enclave-common
  ```
  {: codeblock}

3. Reinstall the SGX driver, which runs the `datashield-sgx` and installs the necessary packages.

Comment (nithyaramakrishnan) [reply ts-index.md]: Hi @smguilia Shawna,

when we restart a worker node it cleans its filesystem. So maybe we can try doing the `purge `first and then the restart of the node. Below is a slight variation to your version:

{: tsCauses}
You might have a library, such as `libsgx-enclave-common`, that is incorrectly installed.

{: tsResolve}
To resolve the issue, try uninstalling the library and then reinstalling the driver.

1. Uninstall the library.
`dpkg --purge libsgx-enclave-common`
{: codeblock}

2. Access the worker node on your cluster and restart it.

3. Reinstall the SGX driver, which runs the `datashield-sgx` and installs the necessary packages.

Thanks,
Nithya

Comment (smguilia) [reply ts-index.md]: Thanks! I pulled it in internally. No need for a new PR. Pushing the change to production now.

**Pull request closed without merging**